scripts/create-layer.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+echo "-----------------------"
+echo "Creating lambda package"
+echo "-----------------------"
+echo
+
+echo "Remove lambda python packages"
+rm -rdf $PREFIX/python/boto3/ \
+  && rm -rdf $PREFIX/python/botocore/ \
+  && rm -rdf $PREFIX/python/docutils/ \
+  && rm -rdf $PREFIX/python/dateutil/ \
+  && rm -rdf $PREFIX/python/jmespath/ \
+  && rm -rdf $PREFIX/python/s3transfer/ \
+  && rm -rdf $PREFIX/python/numpy/doc/
+
+echo "Remove uncompiled python scripts"
+find $PACKAGE_PREFIX -type f -name '*.pyc' | while read f; do n=$(echo $f | sed 's/__pycache__\///' | sed 's/.cpython-36//' | sed 's/.cpython-37//'); cp $f $n; done;
+find $PREFIX/python -type d -a -name '__pycache__' -print0 | xargs -0 rm -rf
+find $PREFIX/python -type f -a -name '*.py' -print0 | xargs -0 rm -f
```

Comment:
> Q: does this work? Does everything get compiled so that removing *.py leaves all corresponding *.pyc files?

there was a bug (now fixed) but yes basically this remove .py and keep only .pyc so the lambda doesn't have to pre-compile the py files

Reply:
> Also found it useful to create two lambda layers, one for the binary libs and another for python packages, so that each layer is smaller (a lambda function can have up to 5 layers, but there is a limit of approx 250Mb on the total size of all layers, last time I read the AWS docs)

Last commit implemented it too

Reply:
>  I found another option/path to smaller results by using the pip option to disable compilation;

@darrenleeweber I'm not sure of the implication of not compiling the `.py` files but I guess that keeping only those afteward make things faster (at start) and lighter too